appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/manager/WalletUtils.java

```diff
@@ -74,10 +78,18 @@ private static void getPackageToBind() {
   }
 
   private static String chooseServiceToBind(List<String> packageNameServices) {
-    String[] packagesOrded = BuildConfig.SERVICE_BIND_LIST.split(",");
-    for (String address : packagesOrded) {
-      if (packageNameServices.contains(address)) {
-        return address;
+    if (isAppInstalled(BuildConfig.CAFE_BAZAAR_PACKAGE_NAME, context.getPackageManager())
```

Comment:
same comment has before: should we check if the user is from Iran **and** has cafe bazaar installed?